nBTC/sources/nbtc.move

```diff
@@ -538,7 +539,13 @@ public fun add_dwallet(
     assert!(!contract.storage.exist(dwallet_id), EDuplicatedDWallet);
 
     let p2wpkh = 0;
```

Comment:
let's use consts. If it's in other module, we can have a function

Reply:
2. how do we know the added wallet is of type p2wpkh?

Reply:
This is presetting by the admin. One more we can use "output.move" to parse the type of output or script.